prow/plugins/trigger/generic-comment.go

```diff
@@ -30,6 +30,7 @@ import (
 var okToTestRe = regexp.MustCompile(`(?m)^/ok-to-test\s*$`)
 var testAllRe = regexp.MustCompile(`(?m)^/test all,?($|\s.*)`)
 var retestRe = regexp.MustCompile(`(?m)^/retest\s*$`)
+var genericTestRe = regexp.MustCompile(`(?m)^/test ($|\s.*)`)
```

Comment:
Stray space, I think.

```suggestion
var genericTestRe = regexp.MustCompile(`(?m)^/test($|\s.*)`)
```

Reply:
On second thought, made this the default that we set in config load and then changed `<name>` with `.*`